src/components/batch-selection/batch-selection.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+
+import BatchSelection from ".";
+import IconButton from "../icon-button";
+import Icon from "../icon";
+import Link from "../link";
+
+export const Default = () => (
+  <BatchSelection selectedCount={0}>
+    <span style={{ paddingRight: "5px" }}>
+      (<Link>Select All 38 items</Link>)
+    </span>
```

Comment:
```suggestion
   <Button size="small" mx={1} buttonType="secondary">
      Select All 38 items
   </Button>
```
What do you think about this to address the lint warning?
![Screenshot 2022-12-12 at 15 24 41](https://user-images.githubusercontent.com/18368713/207084161-1ed7e54f-4aa2-4edd-84a7-213b51016e1f.png)